.gitignore

```diff
@@ -43,3 +43,7 @@ lib*.pc
 /Release/
 /tmp_install/
 /portlock/
+
+# Databricks specific excludes
+.idea
+.idea_modules
```

Comment:
I don't see a good reason to include these - the user can add these to their own `~/.gitignore`.